tasty-inspector/src/scala/tasty/inspector/TastyInspector.scala

```diff
@@ -17,36 +17,29 @@ import dotty.tools.dotc.report
 
 import java.io.File.pathSeparator
 
-trait TastyInspector:
-  self =>
-
-  /** Process a TASTy file using TASTy reflect */
-  protected def processCompilationUnit(using Quotes)(root: quotes.reflect.Tree): Unit
-
-  /** Called after all compilation units are processed */
-  protected def postProcess(using Quotes): Unit = ()
+object TastyInspector:
```

Comment:
Minor: what about just call it `Tasty`? The rationale is to avoid the conceptual link with `Inspector`.

Reply:
We already have trait called `Tasty` that means something different. Maybe we need another name.

Reply:
Then it's fine to keep the current name.